### PR TITLE
Gtw/test_describe

### PR DESCRIPTION
**Pull request by gregertw**
# Description

Added tests for describe datamodel command

## Checklist

- [x] Tests added/updated.
- [ ] Run Demo Job Locally.
- [ ] Documentation updated.
- [ ] Changelogs updated in [CHANGELOG.cdf-tk.md](https://github.com/cognitedata/cdf-project-templates/blob/main/CHANGELOG.cdf-tk.md).
- [ ] Template changelogs updated in [CHANGELOG.templates.md](https://github.com/cognitedata/cdf-project-templates/blob/main/CHANGELOG.templates.md).
- [ ] Version bumped.
  [_version.py](https://github.com/cognitedata/cdf-project-templates/blob/main/cognite/cognite_toolkit/_version.py) and
  [pyproject.toml](https://github.com/cognitedata/cdf-project-templates/blob/main/pyproject.toml) per [semantic versioning](https://semver.org/).

### CODE REVIEW

File: tests/test_cdf_tk/test_describe.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    data_model_loader = DataModelLoader.create_loader(cdf_tool)
+    container_loader = ContainerLoader.create_loader(cdf_tool)
+    view_loader = ViewLoader.create_loader(cdf_tool)
+    spaces = [
+        Space.load(filepath.read_text())
+        for filepath in [
+            file
+            for type_ in space_loader.filetypes
+            for file in Path(DATA_FOLDER, "data_models").glob(f"**/*.{type_}")
+            if re.compile(space_loader.filename_pattern).match(file.stem)
```
Comment (doctrino) [inline tests/test_cdf_tk/test_describe.py:46]: Not in this PR, but maybe this logic should moved into the base Loader.

File: tests/test_cdf_tk/test_describe.py [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    containers = [
+        Container.load(filepath.read_text())
+        for filepath in [
+            file
+            for type_ in container_loader.filetypes
+            for file in Path(DATA_FOLDER, "data_models").glob(f"**/*.{type_}")
+            if re.compile(container_loader.filename_pattern).match(file.stem)
+        ]
+    ]
+    views = View.load(Path(DATA_FOLDER, "data_models", "4.Asset.view.yaml").read_text())
```
Comment (doctrino) [inline tests/test_cdf_tk/test_describe.py:67]: Overwritten in the next line
```suggestion
```

File: tests/test_cdf_tk/test_describe.py [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+    out, _ = capfd.readouterr()
+    try:
+        snapshot = Path(SNAPSHOTS_DIR / "describe_datamodel.txt").read_text()
+        assert (
+            snapshot == out
+        ), f"Snapshot does not match output. Delete {SNAPSHOTS_DIR}/describe_datamodel.txt if you have changed output."
+    except FileNotFoundError:
+        with open(SNAPSHOTS_DIR / "describe_datamodel.txt", "w") as f:
+            f.write(out)
+        assert False, "Snapshot file not found. Created a new one."
```
Comment (doctrino) [inline tests/test_cdf_tk/test_describe.py:92]: Already supported by the `pytest-regression` package we are using. 
```suggestion
    file_regression.check(out, encoding="utf-8", fullpath=SNAPSHOTS_DIR/"describe_datamodel.txt")
```

Comment (doctrino) [reply tests/test_cdf_tk/test_describe.py:92]: Then, the command `--force-regen` will work as well.

**Review by doctrino — COMMENTED** [review]
Just switch to the `file_regression` it is a fixture you can use. Ref https://pytest-regressions.readthedocs.io/en/latest/api.html#file-regression

(note the discussion is a bit outdated as `fullpath` is a supported argument)

**Review by gregertw — COMMENTED** [review]
> Not in this PR, but maybe this logic should moved into the base Loader.  

Yeah, agreed. Suitable for that. I can make a small PR afterwards.

**Review by doctrino — APPROVED** [review]

**Pull request merged**